jquery.flot.touch.js

```diff
@@ -230,8 +239,10 @@
         }
 
         function preventEventPropagation(e) {
-            e.preventDefault();
-            e.stopPropagation();
+            if (!gestureState.allowEventPropagation) {
```

Comment:
So it seams that the touchNavigation, the parent elements and the browser will all handle the event. Is this what we want? Shouldn't we move this check in the interpretGestures() function to make sure we don't pass a new pan or pinch event to the touchNavigation while the browser should handle the current one?